### PR TITLE
Add demographics detail view

### PR DESCRIPTION
**Pull request by JohnTasto**
`src/state/households` and `src/state/populations` are basically simplified versions of `src/state/affordability` and `src/state/rent` - simplified in that the API doesn't take any url parameters. `fetchHouseholdData` and `fetchPopulationsData` are both dispatched only once when `<App />` mounts.

A `neighborhood` property is also added to the `parameters` state to keep track of which neighborhood is selected. Maybe it should be named something else or moved out of `parameters`, but I felt it was a good fit.

Finally, `<DemographicsDetailView />` could probably be broken up into subcomponents, but whatever. It could also use some more styling.

screen shot 2017-04-29 at 12 47 05 am

### CODE REVIEW

**Comment by DingoEatingFuzz** [comment]
Things we still want to do (not blocking merging):

  - Denote which neighborhood is selected
  - Style the "tooltip" so it's the association with the map is clearer
  - Comma-format large numbers.

**Comment by JohnTasto** [comment]
@nmuldavin removed the popup tooltip in #31 - I figured we could draw a thicker outline around the selected neighborhood or something.

**Comment by nmuldavin** [comment]
The wireframe has a pink outline around the selected neighborhood, I think that would look nice

**Comment by nmuldavin** [comment]
Just pulled this up on my machine. I know there was talk about not needing the detail view but so glad you did this work, I think it really rounds it out. Thanks @JohnTasto!

File: src/utils/data-constants.js [hunk 1]
```diff
@@ -22,10 +22,41 @@ export const HOUSING_TYPES = [
...
+  'Raleigh Hills',
+  'Roseway-Cully',
+  'Sellwood-Moreland-Brooklyn',
+  'South Portland-Maraquam Hill',
+  'St. Johns',
+  'Tryon Creek-South Terwilliger',
+  'West Portland',
+  'Woodstock',
+  'Portland',
+];
```
Comment (DingoEatingFuzz) [inline src/utils/data-constants.js]: We should get the name from the data provided to the map instead of re-associating it by means of this constant.

**Review by DingoEatingFuzz — CHANGES_REQUESTED** [review]
Just the one change we talked about in person, this it's good to 🚢 !

File: src/utils/data-constants.js (hunk 1 above)
Comment (JohnTasto) [reply src/utils/data-constants.js]: Done!

**Review by DingoEatingFuzz — APPROVED** [review]

**Pull request merged**